prototype_source/pt2e_quant_x86_inductor.rst

```diff
@@ -154,6 +176,7 @@ Finally, we will convert the calibrated Model to a quantized Model. ``convert_pt
 ::
 
     converted_model = convert_pt2e(prepared_model)
+    torch.ao.quantization.move_exported_model_to_eval(converted_model)
```

Comment:
is this needed for both PTQ and QAT?

Reply:
I feel we could also have a separate section for QAT, like what we did in https://pytorch.org/tutorials/advanced/static_quantization_tutorial.html#quantization-aware-training

Reply:
Thanks for the comment. Yean, I think it will be much clearer for reader. Move the QAT into a separate section. Please kindly help to take a look again @jerryzh168.